cpp/CMakePresets.json

```
@@ -40,6 +40,17 @@
         "CXX": "clang++-15"
       }
     },
+    {
+      "name": "clang15-dbg",
+      "displayName": "Debugging build with Clang-15",
+      "description": "Build with globally installed Clang-15 in debug mode",
+      "inherits": "default",
```

```suggestion
      "inherits": "clang15",
```
I think you can inherit clang15 and then you don't need to re-specify the CC and CXX